content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst

```diff
@@ -1,65 +1,273 @@
+alerts will be created.
+
+.. important::
+   To sell perishable products with expiration dates, the :guilabel:`Removal Strategy` for the
+   :guilabel:`Location` the products are stored in must be set to
+   :abbr:`FEFO (First Expiry, First Out)`. If there is not enough stock of perishable products in
+   one lot, Odoo will automatically take the remaining quantity required from a second lot with the
+   next-soonest expiration date. Removal strategies can also be set on
+   :guilabel:`Product Categories`.
+
+.. seealso::
+   - :doc:`../../routes/strategies/removal`
+
+View expiration dates for lots & serial numbers
+===============================================
+
+To view (and/or group) all products with expiration dates by lot number, go to
+:menuselection:`Inventory --> Products --> Lots/Serial Numbers`.
+
+Once there, remove any default :guilabel:`Filters` in the :guilabel:`Search bar`. Then, click
+:guilabel:`Group By`, :guilabel:`Add Custom Group`, and click the drop-down to select
+:guilabel:`Expiration Date.` Lastly, click :guilabel:`Apply` to apply the filter.
+
+Doing so breaks down all perishable products, their expiration dates, and the assigned lot number.
+
+.. image:: expiration_dates/expiration-dates-group-by-dates.png
+   :align: center
+   :alt: Group by expiration dates on lots and serial numbers page.
+
+Expiration alerts
+-----------------
+
+To see expiration alerts, go to :menuselection:`Inventory --> Products --> Lots/Serial Numbers`.
+
+Then, click into a :guilabel:`Lot/Serial Number` with perishable products. Doing so reveals the
+serial number detail form. On the serial number detail form, click the :guilabel:`Dates` tab to see
+all expiration information related to the products.
+
+To edit the form, click :guilabel:`Edit` in the upper-left corner of the form, then change the
+:guilabel:`Expiration Date` to today's date (or earlier), and :guilabel:`Save` changes.
+
+After saving, the lot number form displays a red :guilabel:`Expiration Alert` at the top of the
+form to indicate that the products in this lot are either expired or expiring soon. From here,
+click back to the :guilabel:`Lots/Serial Numbers` page (via the breadcrumbs).
 
-Expiration Alerts
-=================
+To see the new expiration alert, or any expiration alerts for products that are expired (or will
+expire soon), remove all :guilabel:`Filters` from the :guilabel:`Search bar` on the
+:guilabel:`Lots/Serial Numbers` page.
 
-You can access expiration alerts from the *inventory* app. To do so, go to :menuselection:`Master
-Data --> Lots/Serial Numbers`. There, you can use the pre-existing filter to show all the
-lots/serial numbers that have exceeded their respective alert dates.
+Then, click :guilabel:`Filters`, then :guilabel:`Expiration Alerts`.
 
-.. image:: expiration_dates/expiration_dates_05.png
+.. image:: expiration_dates/expiration-dates-expiration-alert.png
```

Comment:
for all of the images - the filenames currently have the text `expiration-dates-` prepended within each image filename, this is unnecessary since it is already *implied* because these images are within the `expiration_dates` folder.

This principle keeps image filenames concise while still being descriptive.